Translate: intro/thanks.md (#133)
OI-wiki/i18n: docs/intro/thanks.md

disqus:

> This project currently accepting your donations. Scan the QR code below to donate.（Be sure to note:  `‘Donation’ + Your information` or in Chinese: `‘捐赠’+ 自己的信息`）。

Reviewer: ```suggestion
This project currently accepting your donations. Scan the QR code below to donate. (Be sure to remark `‘Donation’ + Your information`)。
```
We have a Chinese site so just English content on this site is okay :)

Author: Ah thank you, I might have forgot when translating because the punctuating mark look the same as English in markdown editor.

> All funds will be used for **OI Wiki** domain、server、all operation and maintenance.

Reviewer: ```suggestion
All funds will be used for **OI Wiki** domain, server, all other operation and maintenance costs.
```

> Donation with large amount will be included at the bottom of this page or in more appropriate place in the future to express gratitute from us.

Reviewer: ```suggestion
Donation with large amount will be included at the bottom of this page or in more appropriate place in the future to express our gratitude.
```
